test/interval_tests/trig_tests.jl

```diff
@@ -75,7 +74,7 @@ facts("Trig tests") do
 
     @fact atan2(∅, entireinterval()) --> ∅
     @fact atan2(entireinterval(), ∅) --> ∅
-    @fact atan2(@interval(0.0, 1.0), @biginterval(0.0)) --> @biginterval(pi/2)
+    @fact atan2(@interval(0.0, 1.0), @biginterval(0.0)) --> @interval(pi/2)
```

Comment:
Why is this change in the tests required? I guess this is related to the fact that
```julia
julia> atan2(@interval(0.0, 1.0), @biginterval(0.0))
[1.5707963267948966, 1.5707963267948968]
julia> typeof(ans)
ValidatedNumerics.Interval{Float64}
```

I do not know where this is induced, but my way of thinking is that, if you use `@biginterval` is because you mean it, so the answer should be a `Interval{BigFloat}`. This is probably related to `convert(Interval,x)`, commented elsewhere.